api/src/main/kotlin/io/quarkus/code/misc/QuarkusExtensionUtils.kt

```diff
@@ -12,7 +12,7 @@ import java.util.concurrent.atomic.AtomicInteger
 
 object QuarkusExtensionUtils {
 
-    val TAG_KEYS = listOf("status", ".+-support", "with")
```

Comment:
remove `.+-support` and instead of adding `-status` suffix for the key, use `support` as suffix, it will be compatible with the current data.

Reply:
no

Reply:
Made suggested changes.